_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+---
+id: be4d0c
+name: Inline link has either distinguishable style not based on color (hue) alone or content that indicates it is a link
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style not based on color (hue) alone or have content that indicates they are links.
+accessibility_requirements:
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
+- **different hue**: the element's [foreground colors][foreground color] and the [foreground color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][], or the element's [background colors][background color] and the [background color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][].
+
+## Expectation
+
+For each [link history state][] with a **different hue** there exists at least one [visible][] [inclusive descendant][] and one [adequate inspection state][] of the test target such that at least one of the following is true:
+
+- **distinguishing style**: the [visible][] [inclusive descendant][] has a [distinguishable style][] from each element containing **non-link line text**; or
```

Comment:
```suggestion
- the [visible][] [inclusive descendant][] has a [distinguishable style][] from each element containing **non-link line text**; or
```

Same thought process a previous comment.

Reply:
I think you need to suggest changes to the rule design instructions 😉

Reply:
I will make an issue and bring it up. But not a blocker for this one.